packages/gestalt/src/icons/chart.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m10.7791936 13.2208064v-13.2208064c-6.05640271.62770002-10.7791936 5.74556089-10.7791936 11.9674117 0 6.6449969 5.38759139 12.0325883 12.0325883 12.0325883 6.2218508 0 11.3397117-4.7227909 11.9674117-10.7791936zm2.5067893-13.2208064c5.6442866.58458325 10.1294339 5.06973052 10.7140171 10.7140171h-10.7140171z" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```

Comment:
@kexin-zhang are we sure these came from PDS designers? These normally dont have the `viewBox="0 0 24 24"` field specified if it came from them.

Reply:
@pfarejowicz The designer I work with sent these over, and I believe they came from PDS. I'll double check and update here